### ISSUE TITLE
TypeError: Cannot call a class as a function

### ISSUE THREAD

**Issue by Dmo16** [tannerlinsley/react-table#1736]
# Using v7

**Describe the bug**
Following example code from the 'basic' table example here: 
[https://github.com/tannerlinsley/react-table/tree/master/examples/basic](url)

When attempting to call the table component I receive this error: 
`    
    TypeError: Cannot call a class as a function

    71 | function _classCallCheck(instance, Constructor) {
    72 |   if (!(instance instanceof Constructor)) {
    73 |     throw new TypeError("Cannot call a class as a function");
    74 |   }
    75 | }

    //Offending line: 
    function Table({ columns, data }) {
`

**To Reproduce**
1. Go to [https://github.com/tannerlinsley/react-table/tree/master/examples/basic](url)
2. Import modules and copy/paste code into your project
3. Start your application

**Expected behavior**
I expect the table to render with the columns and data I am passing it.

The component renders and works if I use V6, but this error occurs with the new v7

**Desktop (please complete the following information):**

- OS: Win10
- Browser chrome
- Version : 78

**Additional context**
As an aside, this import statement from the example docs: 
`import { useTable } from 'react-table'`
Throws error: 
`Attempted import error: 'useTable' is not exported from 'react-table'.`
It only works if I remove the braces.

**Comment by Dmo16** [comment]
Doing more research and debugging here and I'm beginning to think I have a conflict with my node packages between v6 and v7 and that the bug isn't with react-table itself.

**Comment by Dmo16** [comment]
I've done a clean install and now I'm getting this error when trying to start the app: 

`Attempted import error: 'ReactTableDefaults' is not exported from 'react-table'.`

Me thinks I messed something up on my own here.

**Comment by mmorrisgoodway** [comment]
I am having this same exact problem. Did you have any luck? 
I am getting 
```Attempted import error: 'useTable' is not exported from 'react-table'.```

**Comment by Dmo16** [comment]
@mmorrisgoodway I think I may have fixed it, I'm about to test it. Will report back shortly.

**Comment by mmorrisgoodway** [comment]
Thanks a ton. I have a fairly simple app with few dependencies, so it has me a little confused at the moment.

**Comment by mmorrisgoodway** [comment]
I had some luck changing the version to `7.0.0-rc.2`, wiping my node modules, then rebuilding. See if that helps.

**Comment by Dmo16** [comment]
Okay @mmorrisgoodway I fixed my own issue, not sure if it'll help with your issue. Here's what I did: 

[https://github.com/tannerlinsley/react-table#i-want-to-migrate-from-v6-to-v7-how-do-i-do-that](url)

The second paragraph Tanner talks about having both versions running together, V6 and V7. 

I kept trying to simply remove v6 and run v7 but my project had some dependencies setup on v6 that I wasn't aware of.

So I uninstalled v6, installed `react-table-6` as per Tanner's instructions in the last paragraph there, and then installed v7 again with `yarn add react-table@7`.

This will give you a dialog asking you which version of 7 to install, I simply chose the latest release candidate which is rc3. 

So: `7.0.0-rc3`

The hook import now works perfectly fine, and I'm able to use v6 tables as well as v7.

Probably not a great idea to mix and match in a project if you're starting out but my case the framework I'm using had dependencies tied to v6 that I didn't know I was breaking.

So in conclusion, aliasing v6 fixed my issues.

Please let me know if you need clarification!

**Comment by tannerlinsley** [comment]
There is also an issue with 6.x releases being tagged with the @latest tag when they shouldn't be. This should be fixed within the next few hours.

**Comment by Dmo16** [comment]
> There is also an issue with 6.x releases being tagged with the @latest tag when they shouldn't be. This should be fixed within the next few hours.

No problem! It explains why my first few issues were happening certainly. Each time I re-installed it seemed to be referencing v6. 

It wasn't until I found your migration fix that I discovered I also had dependencies to solve.

Thanks for the great work, very useful library!

**Comment by tannerlinsley** [comment]
This should be fixed now.
